src/components/ui/health-indicator.tsx

```
@@ -0,0 +1,198 @@
+    </div>
+  );
+}
+
+function getContainerClasses(size: string): string {
+  switch (size) {
+    case 'sm':
+      return 'p-1';
+    case 'lg':
+      return 'p-3';
+    case 'md':
+    default:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)